migrating_from_ocp_3_to_4/index.adoc

```diff
@@ -0,0 +1,81 @@
+[id="migration-from-version-3-to-4-overview"]
+= Migration from version 3 to 4 overview
+include::modules/common-attributes.adoc[]
+:context: migration-from-version-3-to-4-overview
+
+toc::[]
+
+{product-title} 4 clusters are different from {product-title} 3 clusters. {product-title} 4 clusters contain new technologies and functionality that result in a cluster that is self-managing, flexible, and automated. To learn more about migrating from {product-title} 3 to 4 see xref:../migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc#about-migrating-from-3-to-4[About migrating from OpenShift Container Platform 3 to 4].
+
+[id="mtc-3-to-4-overview-differences-mtc"]
+=== Differences between {product-title} 3 and 4
+Before migrating from {product-title} 3 to 4, you can check xref:../migrating_from_ocp_3_to_4/planning-migration-3-4.adoc#planning-migration-3-4[differences between {product-title} 3 and 4]. Review the following information:
+
+** Architecture
```

Comment:
They appear to be rendering correctly, but some of your [bulleted lists](https://docs.asciidoctor.org/asciidoc/latest/lists/unordered/#nested-unordered-list) are nested one level deeper than they need to be.

Reply:
ack